web-components/src/components/form/CreditRetireForm.tsx

```diff
@@ -90,41 +95,79 @@ const sxs = {
 
 export interface BottomCreditRetireFieldsProps {
   mapboxToken: string;
+  arrayPrefix?: string;
+  arrayIndex?: number;
 }
 
+type LocationType = {
+  country?: string;
+  stateProvince?: string;
+  postalCode?: string;
+};
+
 export const BottomCreditRetireFields: React.FC<BottomCreditRetireFieldsProps> =
-  ({ mapboxToken }) => {
+  ({ mapboxToken, arrayPrefix = '', arrayIndex }) => {
     const styles = useStyles();
-    const {
-      values: { country, stateProvince, postalCode },
-      setFieldValue,
-    } = useFormikContext<RetireFormValues>();
+    const { values, setFieldValue } = useFormikContext<
+      RetireFormValues | RetireFormValuesArray
+    >();
+    const [geocodingError, setGeocodingError] = useState<boolean>(false);
```

Comment:
I believe the best would be to set this as an error msg that then can be used to be displayed in the error banner (see suggestion below)